confidential/confidential_test.go

```diff
@@ -79,3 +105,92 @@ func TestUnblindOutput(t *testing.T) {
 		assert.Equal(t, hex.EncodeToString(output.ValueBlindingFactor[:]), valueBlindingFactor)
 	}
 }
+
+func TestFinalValueBlindingFactor(t *testing.T) {
+	err := setUp()
+	if !assert.NoError(t, err) {
+		t.FailNow()
+	}
+
+	vectors := tests["FinalValueBlindingFactorTC"].([]interface{})
+	for _, testVector := range vectors {
+		v := testVector.(map[string]interface{})
+
+		inValuesSlice := v["inValues"].([]interface{})
+		inValues := make([]uint64, 0)
+		for _, v := range inValuesSlice {
+			n, err := strconv.ParseUint(fmt.Sprintf("%v", v), 10, 64)
```

Comment:
why not `v.(string)` instead of `fmt.Sprintf()`?

Reply:
Done